packages/pharos/src/components/link/pharos-link.test.ts

```diff
@@ -15,6 +15,12 @@ describe('pharos-link', () => {
     await expect(component).to.be.accessible();
   });
 
+  it('is accessible as a button', async () => {
+    component.href = undefined;
```

Comment:
```suggestion
    component.removeAttribute('href');
```

Reply:
nice. Fixed it. Resolving this conversation.